examples/jaeger-tracing/front-envoy-jaeger.json

```diff
@@ -1,84 +1,107 @@
 {
-  "listeners": [
-    {
-      "address": "tcp://0.0.0.0:80",
-      "filters": [
-        {
-          "name": "http_connection_manager",
-          "config": {
-            "tracing": {
-              "operation_name": "egress"
-            }, 
-            "codec_type": "auto",
-            "stat_prefix": "ingress_http",
-            "route_config": {
-              "virtual_hosts": [
-                {
-                  "name": "backend",
-                  "domains": ["*"],
-                  "routes": [
-                    {
-                      "prefix": "/",
-                      "cluster": "service1",
-                      "decorator": {
-                        "operation": "checkAvailability"
+  "static_resources": {
```

Comment:
QQ: do we want to switch to YAML as well for human readable examples when doing the v2 conversion?

Reply:
I would prefer YAML, but I didn't want to ask @mccv to do that extra work. Is there a quick way to convert JSON to YAML via tool?